.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+	"image": "mcr.microsoft.com/vscode/devcontainers/python:3",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				// Settings for the python extension.
+				"python.defaultInterpreterPath": "${workspaceFolder}/.venv",
+				"python.terminal.activateEnvInCurrentTerminal": true,
+				"python.linting.enabled": true,
+				"python.linting.pylintEnabled": true
+
+			},
+			// Installs extensions on container create.
+			"extensions": [
+				"ms-python.python",
+				"ms-python.vscode-pylance"
+			]
+		}
+	},
+
+	// Displays a custom welcome message.
+	"onCreateCommand": "sudo cp .devcontainer/welcome-message.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt",
```

Comment:
Suggestion: You could avoid adding `onCreateCommand` if you move this command to the Dockerfile.

Reply:
Nevermind, I accidentally thought this PR had a Dockerfile